tests/test_annotation_tilerendering.py

```diff
@@ -59,21 +59,21 @@ def _fill_store(
         cells = [
             Annotation(
                 cell,
-                {"type": "cell", "prob": RNG.random(1)[0], "color": (0, 1, 0)},
+                {"type": "cell", "prob": RNG.random(1)[0], "colour": (0, 1, 0)},
```

Comment:
Big fan of the u 😆

Reply:
We are using US spellings in the toolbox :)